Jenkins_jobs/WorkspaceInfo.groovy

```diff
@@ -3,136 +3,96 @@ import jenkins.model.Jenkins;
 import hudson.model.Computer;
 
 clones = [:]
-List<String> newMachines = new ArrayList<String>();
 
 /* Iterates over all the online nodes in Jenkins and
  * prints contents of workspace folder along with
  * the space they occupy. With exception of tmp directory
+ * tmp directory is any directory with "@tmp" in the name
+ * 
+ * The nodes can be either passed in or grabbed from Jenkins
  */
 node {
```

Comment:
I don't think you understood my previous question. Do we need a node block to do this?